python/ray/tune/tune.py

```diff
@@ -142,6 +163,16 @@ def _report_progress(
         reporter.report(trials, done, sched_debug_str, executor_debug_str)
 
 
+def _report_air_progress(
+    runner: TrialRunner, reporter: "AirProgressReporter", force: bool = False
+):
+    trials = runner.get_trials()
+    reporter_args = []
+    executor_debug_str = runner.trial_executor.debug_string()
+    reporter_args.append(executor_debug_str)
```

Comment:
```suggestion
    used_resources_str = runner._used_resources_string()
    reporter_args.append(used_resources_str)
```
for compatibility with our latest refactor :-)

Reply:
ah thanks for pointing that out!